burn-import/src/burn/node/unary.rs

```diff
@@ -0,0 +1,85 @@
+use super::{Node, NodeCodegen};
+use crate::burn::{Scope, TensorType, ToTokens, Type};
+use burn::record::PrecisionSettings;
+use proc_macro2::TokenStream;
+use quote::quote;
+use std::sync::Arc;
+
+// Simple fn pointer that receive input as a token stream and return function call.
+type FnPointer = Arc<dyn Fn(TokenStream) -> TokenStream>;
+
+/// Node for all unary operators.
+#[derive(Clone, new)]
+pub struct UnaryNode {
+    pub input: TensorType,
+    pub output: TensorType,
+    pub name: String,
```

Comment:
Can we define an enum instead of using a String?

Reply:
Sounds like a good idea.